packages/editor/src/client/panel/mapLeafletEditPanel.tsx

```diff
@@ -0,0 +1,213 @@
+
+  return (
+    <>
+      <Box display="flex" flexDirection="column">
+        <AddressInput
+          label="Address"
+          value={[{address: address, lat: lat, lng: lng}]}
+          marginBottom={Spacing.ExtraSmall}
+          onChange={markerPoints => {
+            if (markerPoints !== undefined && markerPoints.length > 0) {
+              onChange({
+                ...value,
+                address: markerPoints[1].address,
+                lat: markerPoints[1].lat,
+                lng: markerPoints[1].lng
+              })
+            }
+          }}
+        />
+        <TextInput
+          marginBottom={Spacing.ExtraSmall}
+          type="number"
+          label="Latitude"
+          value={lat}
+          onChange={e => onChange({...value, lat: parseInt(e.target.value)})}
+          required
+        />
+        <TextInput
+          marginBottom={Spacing.ExtraSmall}
+          type="number"
+          label="Longitude"
+          value={lng}
+          onChange={e => onChange({...value, lng: parseInt(e.target.value)})}
+          required
+        />
+        <TextInput
+          marginBottom={Spacing.ExtraSmall}
+          label="Title"
+          value={title}
+          onChange={e => onChange({...value, title: e.target.value})}
+        />
+        <TextInput
+          marginBottom={Spacing.ExtraSmall}
+          label="Description"
+          value={description}
+          onChange={e => onChange({...value, description: e.target.value})}
+        />
+      </Box>
+    </>
+  )
+}
+
+export interface AddressInputProps extends MarginProps {
+  label?: string
+  description?: string
+  value: MarkerPoint[]
+  onChange(address?: MarkerPoint[]): void
+}
+
+export function AddressInput(props: AddressInputProps) {
+  return (
+    <AutocompleteInput
+      {...props}
+      label={props.label}
+      valueToChipData={markerPoint => ({
+        id: nanoid(),
+        label: markerPoint.address
+      })}>
+      {propsAutocompleteInput => <AddressInputList {...propsAutocompleteInput} />}
+    </AutocompleteInput>
+  )
+}
+
+function AddressInputList({
+  isOpen,
+  inputValue,
+  highlightedIndex,
+  getItemProps,
+  getMenuProps
+}: AutocompleteInputListProps) {
+  const [items, setItems] = useState<MarkerPoint[]>([])
+  const api = 'ab48a007c6dd4906adcd2871a5deaebe'
+  const fetchURL = `https://api.opencagedata.com/geocode/v1/json?key=${api}&q=${inputValue}&limit=5&pretty=1`
+
+  useEffect(() => {
+    async function fetchData() {
+      if (inputValue !== null && inputValue.length > 1) {
+        try {
+          const request = await axios.get(fetchURL)
+          const items = request.data.results.map((res: any) => {
+            return {
+              lat: res.geometry.lat,
+              lng: res.geometry.lng,
+              address: res.formatted
+            }
+          })
+          setItems(items)
+        } catch (error) {
+          console.error(error)
```

Comment:
That's not what you do with errors. Either display a message to the user or handle it.

Reply:
Like [that](https://gist.github.com/fgilio/230ccd514e9381fafa51608fcf137253) for example?